parsons/databases/db_sync.py

```diff
@@ -373,3 +409,95 @@ def create_table(self, source_table, destination_table):
                 "Unable to create destination table based on source table; we will "
                 'fallback to using "copy" to create the destination.'
             )
+
+    def upsert_rows(
+        self,
+        source_table_name: str,
+        destination_table_name: str,
+        cutoff,
```

Comment:
type?

Reply:
Help me think through this? 

string works, numbers of any kind work, datetime works - really any type of python object that is going to be translated from a SQL query by parsons that can be compared with < , > , == 

I guess we'd want to look through all possible python objects that parsons might make from queries from any of the subclasses of parsons.databases.database_connector.DatabaseConnector

probably `Union[int, float, str, datetime.datetime, datetime.date]` would cover most or all of the cases - but I wasn't sure so I skipped it out of laziness and caution

Reply:
Yeah, that makes sense. I'd say, since it's not binding, we go with your suggestion that covers all the cases we can think of. It's at least a starting place for anyone looking into things, and if someone uses a type that isn't listed they can always submit a small PR.

Reply:
Done with 91db890